app/models/work.rb

```diff
@@ -628,6 +632,13 @@ def adjust_series_restriction
     end
   end
 
+  # Save relationship to series if applicable
+  def save_series
```

Comment:
`series_attributes` already sets up a series and associating it with the work, based on info extracted from `params`. I think you should be able to remove `save_series` completely and everything would still work.